Updated README with new schedule and time

PieterKas opened the pull request:
Updated README with new schedule and time

PieterKas committed:
Updated README with new schedule and time

Signed-off-by: Pieter Kasselman <pieter@spirl.com>

evan2645 commented on community/sig-spec/README.md:
```diff
@@ -3,7 +3,7 @@
 
 ### Meetings:
-* Date/Time: Every Thursday @ 1:00pm Pacific
+* Date/Time: Every Second Thursday @ 9:30AM PST
```
```suggestion
* Date/Time: Every other Thursday @ 9:30AM PST
```

I think as written it could be misinterpreted as once per month

amoore877 committed:
fix up voter script (#325)

* fix up voter script

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>

* complete message

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>

---------

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>
Signed-off-by: Pieter Kasselman <pieter@spirl.com>

amoore877 committed:
improve election issue template wording and organization (#326)

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>
Signed-off-by: Pieter Kasselman <pieter@spirl.com>

dfeldman committed:
Add Volkan Ozcelik as nominee (#329)

Signed-off-by: Daniel Feldman <dfeldman.mn@gmail.com>
Signed-off-by: Pieter Kasselman <pieter@spirl.com>

faisal-memon committed:
Add Mariusz Sabath as nominee (#331)

Signed-off-by: Faisal Memon <fymemon@yahoo.com>
Signed-off-by: Pieter Kasselman <pieter@spirl.com>

fkautz committed:
Add Frederick Kautz nomination for 2025H1 SSC election (#332)

Signed-off-by: Frederick F. Kautz IV <fkautz@alumni.cmu.edu>
Signed-off-by: Pieter Kasselman <pieter@spirl.com>

amoore877 committed:
2025H1 election results (#338)

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>
Signed-off-by: Pieter Kasselman <pieter@spirl.com>

arndt-s commented:
@PieterKas can we get this merged? Feel free to also give me access to your branch and I can work in the suggested changes.

kfox1111 committed:
Open Nominations for 2025 H1 SSC Election (#324)

* Open Nominations for 2025 H1 SSC Election

Signed-off-by: Kevin Fox <Kevin.Fox@pnnl.gov>

* Set tracking issue

Signed-off-by: Kevin Fox <Kevin.Fox@pnnl.gov>

---------

Signed-off-by: Kevin Fox <Kevin.Fox@pnnl.gov>
Co-authored-by: amoore877 <11066940+amoore877@users.noreply.github.com>
Signed-off-by: Pieter Kasselman <pieter@spirl.com>

heymarcel committed:
Add SPIRL to vendor-technical-contacts (#328)

* Add SPIRL to vendor-technical-contacts folder

Following the format established in
https://github.com/spiffe/spiffe/pull/298

Signed-off-by: Marcel Levy <marcel@spirl.com>

* Add Teleport to vendor-technical-contacts folder (#298)

* Add Teleport to vendor-technical-contacts folder

Signed-off-by: Noah Stride <noah.stride@goteleport.com>

* Replace ampersand with "and"

Signed-off-by: Noah Stride <noah.stride@goteleport.com>

---------

Signed-off-by: Noah Stride <noah.stride@goteleport.com>
Signed-off-by: Marcel Levy <marcel@spirl.com>

* Introduce vendor contact template

Introducing a template for vendor contacts to align information.

Signed-off-by: Volkan Özçelik <volkan.ozcelik@broadcom.com>
Signed-off-by: Marcel Levy <marcel@spirl.com>

* submit formal proposals to GH (#302)

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>
Signed-off-by: Marcel Levy <marcel@spirl.com>

* cosmetic: minor grammar fix (#303)

Signed-off-by: Volkan Özçelik <volkan.ozcelik@broadcom.com>
Signed-off-by: Marcel Levy <marcel@spirl.com>

* expanded code of conduct definitions and process (#300)

* expanded code of conduct definitions and process

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>

* wording

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>

* retaliation wording

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>

* frequent->excessive

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>

* smithing

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>

* unilateral update

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>

---------

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>
Signed-off-by: Marcel Levy <marcel@spirl.com>

* add SSC reporting form for CoC (#305)

Looks good; landing.

--
Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>
Signed-off-by: Marcel Levy <marcel@spirl.com>

* Introduce SPIFFE Bundle Map spec (#304)

* Introduce SPIFFE Bundle Map spec

This PR introduces the SPIFFE Bundle Map text that has been making its
way through SIG-Spec. A link to that document is below for posterity.

https://docs.google.com/document/d/13KHycYIfRC-g42aEfo4_4inF_WauCXOBgZAcUGuxdgs

Signed-off-by: Evan Gilman <evan@spirl.com>

* Editorial and formatting fixes to SPIFFE bundle map doc

Signed-off-by: Evan Gilman <evan@spirl.com>

* Fix typo on trust domain and bundle spec

Signed-off-by: Evan Gilman <evan@spirl.com>

---------

Signed-off-by: Evan Gilman <evan@spirl.com>
Signed-off-by: Marcel Levy <marcel@spirl.com>

* Editorial changes to X509-SVID (#316)

* editorial changes to X509-SVID

Signed-off-by: Arndt <17650715+arndt-s@users.noreply.github.com>

* remove outdated sentence regarding percent-encoding of SPIFFE-ID

Signed-off-by: Arndt <17650715+arndt-s@users.noreply.github.com>

---------

Signed-off-by: Arndt <17650715+arndt-s@users.noreply.github.com>
Signed-off-by: Marcel Levy <marcel@spirl.com>

* Update Teleport documentation link in `vendor-technical-contacts` (#318)

Signed-off-by: Noah Stride <noah.stride@goteleport.com>
Signed-off-by: Marcel Levy <marcel@spirl.com>

* add financials documentation (#314)

* add financials documentation

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>

* finish instructions

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>

* appreciation list

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>

---------

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>
Signed-off-by: Marcel Levy <marcel@spirl.com>

* fix up voter script (#325)

* fix up voter script

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>

* complete message

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>

---------

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>
Signed-off-by: Marcel Levy <marcel@spirl.com>

* improve election issue template wording and organization (#326)

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>
Signed-off-by: Marcel Levy <marcel@spirl.com>

* Add SPIRL to vendor-technical-contacts folder

Signed-off-by: Marcel Levy <marcel@spirl.com>

---------

Signed-off-by: Marcel Levy <marcel@spirl.com>
Signed-off-by: Noah Stride <noah.stride@goteleport.com>
Signed-off-by: Volkan Özçelik <volkan.ozcelik@broadcom.com>
Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>
Signed-off-by: Evan Gilman <evan@spirl.com>
Signed-off-by: Arndt <17650715+arndt-s@users.noreply.github.com>
Co-authored-by: Noah Stride <noah@noahstride.co.uk>
Co-authored-by: Volkan Özçelik <volkan.ozcelik@broadcom.com>
Co-authored-by: amoore877 <11066940+amoore877@users.noreply.github.com>
Co-authored-by: Volkan Özçelik <me@volkan.io>
Co-authored-by: Evan Gilman <evan@spirl.com>
Co-authored-by: Arndt Schwenkschuster <17650715+arndt-s@users.noreply.github.com>
Co-authored-by: kfox1111 <Kevin.Fox@pnnl.gov>
Signed-off-by: Pieter Kasselman <pieter@spirl.com>

amoore877 committed:
set up durable history for project proposals (#294)

* set up durable history for project proposals

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>

* google doc optional

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>

* specific details header callout

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>

---------

Signed-off-by: andrew.s.moore <andrew.s.moore@uber.com>
Signed-off-by: Pieter Kasselman <pieter@spirl.com>

arndt-s committed:
fix typo (#341)

Signed-off-by: Pieter Kasselman <pieter@spirl.com>

PieterKas committed:
Update community/sig-spec/README.md

Co-authored-by: Evan Gilman <evan2645@gmail.com>
Signed-off-by: Pieter Kasselman <pieter@spirl.com>

PieterKas committed:
Merge branch 'main' into update-readme

PieterKas commented:
@evan2645 I accepted the proposed changes - can you review and approve so this can be merged?

kfox1111 reviewed (approved):
LGTM. Thanks! :)
